api/discovery.capnp

```diff
@@ -0,0 +1,41 @@
+using Go = import "/go.capnp";
+
+@0xfcba4f486a351ac3;
+
+$Go.package("discovery");
+$Go.import("github.com/wetware/ww/internal/api/discovery");
+
+
+interface DiscoveryService {
+    provider @0 (name :Text) -> (provider :Provider);
+    locator @1 (name :Text) -> (locator :Locator);
+}
+
+interface Provider {
+    provide @0 (addrs :Addr) -> ();
+}
+
+interface Locator {
+    findProviders @0 (chan :Sender(Addr)) -> ();
+    
+    using Sender = import "channel.capnp".Sender;
```

Comment:
I wonder if it's legal to write `using Sender = import "channel.capnp".Sender(Addr);`, or similar 🤔